CollegeMatchWebApp/src/main/java/School.java

```diff
@@ -909,4 +927,85 @@ public int getId() {
 	public void setId(int id) {
 		this.id = id;
 	}
+
+	public int getPopProg1ID() {
+		return popProg1ID;
+	}
+
+	public void setPopProg1ID(int popProg1ID) {
+		this.popProg1ID = popProg1ID;
+	}
+
+	public boolean isPopProg1IDIsNotNull() {
```

Comment:
Can you rename these along the lines of isPopProg1IDNotNull? The extra _is_ that eclipse has in the autogenerated names really bothers me haha.